floodviz/static/js/hydrograph_onready.js

```diff
@@ -0,0 +1,51 @@
+document.addEventListener('DOMContentLoaded', function (event) {
+	'use strict';
+
+	// Hydrograph options
+	var hydro_options = {
+		'disableInteractions': true,
+		'height': FV.hydrograph_dimensions.height,
+		'width': FV.hydrograph_dimensions.width,
+		'display_ids': FV.hydrograph_display_ids,
+		'div_id': '#hydrograph',
+		'watermark_path_1': 'm234.95 15.44v85.037c0 17.938-10.132 36.871-40.691 36.871-27.569 0-40.859-14.281-40.859-' +
```

Comment:
Something to think about but you don't need to do now is putting the watermark string in a separate javascript file so that we don't have to include the whole string in two different files.